### PR TITLE
3.2.0

### PR DESCRIPTION
**Pull request by zzak**
Changes include:

* Expose `working_dir` to options: 13ac8b and b2a60d1
* Replace any non-alpha chars with underscores: 85389e3 (test: 5be98cb)
* Change output message when generating test report:  b45d257
* Add require loader for `gem 'minitest-ci'` to autoload "minitest/ci": 750ae32

Plus many documentation fixes.

/cc @nwjsmith

### CODE REVIEW

File: lib/minitest/ci.rb [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
   class Ci < Reporter
-
-    VERSION = '3.1.0'
-
     class << self
 
       ##
       # Accessor method to change the report dir if you don't like the
-      # defaults.
+      # defaults. (defaults to "test/reports")
```
Comment (nwjsmith) [inline lib/minitest/ci.rb]: To be consistent with the `:working_dir` docs below, how about:
```ruby
##
# Change the report directory. (defaults to "test/reports")
```
?

File: lib/minitest/ci.rb [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
 
       attr_accessor :report_dir
 
       ##
       # Clean the report_dir between test runs? (defaults to true)
 
       attr_accessor :clean
+
+      ##
+      # Change the working direcotry. (defaults to `$PWD`)
```
Comment (nwjsmith) [inline lib/minitest/ci.rb]: `direcotry` -> `directory`

File: lib/minitest/ci.rb [hunk 3]
```diff
@@ -48,14 +48,16 @@ def record result
...
 
+    ##
+    # Generate test report
     def report
       io.puts
-      io.puts 'generating ci files'
+      io.puts '[Minitest::CI] Generating test report in JUnit XML format...'
 
       Dir.chdir report_dir do
         results.each do |name, resultz|
```
Comment (nwjsmith) [inline lib/minitest/ci.rb]: Should this be `results.each do |name, result|`? Not sure why the block param is named `resultz`...

**Review by nwjsmith — CHANGES_REQUESTED** [review]

File: .ruby-version [hunk 4]
```diff
@@ -1 +1 @@
-2.3.1
+2.3.3
```
Comment (nwjsmith) [inline .ruby-version]: When do you think it'd be a good time to switch this to 2.4.0?

File: lib/minitest/ci.rb [hunk 5]
```diff
@@ -6,8 +6,7 @@ class Ci < Reporter
     class << self
 
       ##
-      # Accessor method to change the report dir if you don't like the
-      # defaults. (defaults to "test/reports")
+      # Change the report directory # defaults. (defaults to "test/reports")
```
Comment (nwjsmith) [inline lib/minitest/ci.rb]: A wild octothorpe appears! I think the ` # ` in the middle should be ✂️

**Review by nwjsmith — APPROVED** [review]

**Comment by nwjsmith** [comment]
💯

**Comment by zzak** [comment]
https://rubygems.org/gems/minitest-ci/versions/3.2.0

**Pull request merged**